test/e2e/checkpoint_test.go

```diff
@@ -1218,25 +1213,18 @@ var _ = Describe("Podman checkpoint", func() {
 			result.WaitWithDefaultTimeout()
 			Expect(result).To(ExitCleanly())
 
-			session = podmanTest.Podman([]string{
-				"pod",
-				"create",
-				"--share",
-				share,
-			})
+			session = podmanTest.Podman([]string{"pod", "create", "--name", podName, "--share", share})
 			session.WaitWithDefaultTimeout()
 			Expect(session).To(ExitCleanly())
 			podID = session.OutputToString()
 
-			// Restore container with different port mapping
-			result = podmanTest.Podman([]string{
-				"container",
-				"restore",
-				"--pod",
-				podID,
-				"-i",
-				fileName,
-			})
+			// Restore container into Pod.
+			// Verify that restore works with both Pod name and ID.
+			if index%2 == 1 {
+				result = podmanTest.Podman([]string{"container", "restore", "--pod", podName, "-i", fileName})
+			} else {
+				result = podmanTest.Podman([]string{"container", "restore", "--pod", podID, "-i", fileName})
+			}
```

Comment:
This is 80% duplication. Would you consider cleaning up so the variable part is the only difference?
```go
    podArg := podName
    if index%2 == 1 {
        podArg = podID
    }
    result = ... podArg,
```

Reply:
@edsantiago Thank you for the suggestion! I've updated the pull request with this change.